apps/designer/app/shared/nano-states/nano-states.ts

```diff
@@ -23,6 +23,16 @@ export const useSetPresetStyles = (presetStyles?: PresetStyles) => {
   });
 };
 
+export const stylesContainer = createValueContainer<Styles>([]);
+export const useStyles = () => useValue(stylesContainer);
+export const useSetStyles = (styles?: Styles) => {
+  useState(() => {
+    if (styles) {
```

Comment:
what happens if styles is undefined? don't we want to unset?

Reply:
This is just type guard. It should not be possible.

Reply:
if it should be impossible, then types should not define it as optional, but if its possible then maybe you wanna allow setting an undefined